Tasks/ServiceFabricComposeDeploy/task.json

```diff
@@ -174,4 +174,4 @@
         "CheckingComposeFile": "Checking compose file",
         "UsingPreviewAPI": "Using the preview APIs"
     }
-}
```

Comment:
whats this change?

Reply:
newline is trimmed now. my VSCode does automatic trimming. I don't think this will cause any problem.